### PR TITLE
Fix did resolution context

### PR DESCRIPTION
**Pull request by OR13**

### CODE REVIEW

File: packages/core/src/DocumentComposer.ts [hunk 1]
```diff
@@ -147,7 +147,7 @@ export default class DocumentComposer {
     }
 
     const didResolutionResult: any = {
-      '@context': 'https://www.w3.org/ns/did-resolution/v1',
+      '@context': 'https://w3id.org/did-resolution/v1',
```
Comment (gjgd) [inline packages/core/src/DocumentComposer.ts:150]: This fixes the URL, but now the context doesnt match the name of the properties.

See: https://jsonld-checker.com/?tab=1&url=https%3A%2F%2Fgithub.com%2Ftransmute-industries%2Fsidetree.js%2Fblob%2Fcacf528feffb32a66b99fd8025df75cd05a78a94%2Fpackages%2Ftest-vectors%2Fsrc%2Fdif-sidetree-test-vectors%2Fupdate%2FresultingDocument.json

**Review by gjgd — CHANGES_REQUESTED** [review]

File: packages/core/src/DocumentComposer.ts (hunk 1 above)
Comment (OR13) [reply packages/core/src/DocumentComposer.ts:150]: See https://github.com/decentralized-identity/sidetree/blob/master/tests/vectors/resolution/afterUpdate.json#L60
```
"didDocumentMetadata": {
    "canonicalId": "did:sidetree:EiDyOQbbZAa3aiRzeCkV7LOx3SERjjH93EXoIM3UoN4oWg",
    "method": {
      "published": true,
      "recoveryCommitment": "EiBfOZdMtU6OBw8Pk879QtZ-2J-9FbbjSZyoaA_bqD4zhA",
      "updateCommitment": "EiDOrcmPtfMHuwIWN6YoihdeIPxOKDHy3D6sdMXu_7CN0w"
    }
  }
```

Comment (OR13) [reply packages/core/src/DocumentComposer.ts:150]: hmm [sidetree-core](https://www.jsonld-checker.com/?json=%257B%250A%2520%2520%2522%2540context%2522%253A%2520%2522https%253A%252F%252Fw3id.org%252Fdid-resolution%252Fv1%2522%252C%250A%2520%2520%2522didDocument%2522%253A%2520%257B%250A%2520%2520%2520%2520%2522id%2522%253A%2520%2522did%253Asidetree%253AEiDyOQbbZAa3aiRzeCkV7LOx3SERjjH93EXoIM3UoN4oWg%2522%252C%250A%2520%2520%2520%2520%2522%2540context%2522%253A%2520%255B%250A%2520%2520%2520%2520%2520%2520%2522https%253A%252F%252Fwww.w3.org%252Fns%252Fdid%252Fv1%2522%252C%250A%2520%2520%2520%2520%2520%2520%257B%250A%2520%2520%2520%2520%2520%2520%2520%2520%2522%2540base%2522%253A%2520%2522did%253Asidetree%253AEiDyOQbbZAa3aiRzeCkV7LOx3SERjjH93EXoIM3UoN4oWg%2522%250A%2520%2520%2520%2520%2520%2520%257D%250A%2520%2520%2520%2520%255D%252C%250A%2520%2520%2520%2520%2522service%2522%253A%2520%255B%250A%2520%2520%2520%2520%2520%2520%257B%250A%2520%2520%2520%2520%2520%2520%2520%2520%2522id%2522%253A%2520%2522%2523service1Id%2522%252C%250A%2520%2520%2520%2520%2520%2520%2520%2520%2522type%2522%253A%2520%2522service1Type%2522%252C%250A%2520%2520%2520%2520%2520%2520%2520%2520%2522serviceEndpoint%2522%253A%2520%2522http%253A%252F%252Fwww.service1.com%2522%250A%2520%2520%2520%2520%2520%2520%257D%250A%2520%2520%2520%2520%255D%252C%250A%2520%2520%2520%2520%2522verificationMethod%2522%253A%2520%255B%250A%2520%2520%2520%2520%2520%2520%257B%250A%2520%2520%2520%2520%2520%2520%2520%2520%2522id%2522%253A%2520%2522%2523publicKeyModel1Id%2522%252C%250A%2520%2520%2520%2520%2520%2520%2520%2520%2522controller%2522%253A%2520%2522%2522%252C%250A%2520%2520%2520%2520%2520%2520%2520%2520%2522type%2522%253A%2520%2522EcdsaSecp256k1VerificationKey2019%2522%252C%250A%2520%2520%2520%2520%2520%2520%2520%2520%2522publicKeyJwk%2522%253A%2520%257B%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2522kty%2522%253A%2520%2522EC%2522%252C%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2522crv%2522%253A%2520%2522secp256k1%2522%252C%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2522x%2522%253A%2520%2522tXSKB_rubXS7sCjXqupVJEzTcW3MsjmEvq1YpXn96Zg%2522%252C%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2522y%2522%253A%2520%2522dOicXqbjFxoGJ-K0-GJ1kHYJqic_D_OMuUwkQ7Ol6nk%2522%250A%2520%2520%2520%2520%2520%2520%2520%2520%257D%250A%2520%2520%2520%2520%2520%2520%257D%252C%250A%2520%2520%2520%2520%2520%2520%257B%250A%2520%2520%2520%2520%2520%2520%2520%2520%2522id%2522%253A%2520%2522%2523additional-key%2522%252C%250A%2520%2520%2520%2520%2520%2520%2520%2520%2522controller%2522%253A%2520%2522%2522%252C%250A%2520%2520%2520%2520%2520%2520%2520%2520%2522type%2522%253A%2520%2522EcdsaSecp256k1VerificationKey2019%2522%252C%250A%2520%2520%2520%2520%2520%2520%2520%2520%2522publicKeyJwk%2522%253A%2520%257B%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2522kty%2522%253A%2520%2522EC%2522%252C%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2522crv%2522%253A%2520%2522secp256k1%2522%252C%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2522x%2522%253A%2520%2522aN75CTjy3VCgGAJDNJHbcb55hO8CobEKzgCNrUeOwAY%2522%252C%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2522y%2522%253A%2520%2522K9FhCEpa_jG09pB6qriXrgSvKzXm6xtxBvZzIoXXWm4%2522%250A%2520%2520%2520%2520%2520%2520%2520%2520%257D%250A%2520%2520%2520%2520%2520%2520%257D%250A%2520%2520%2520%2520%255D%252C%250A%2520%2520%2520%2520%2522authentication%2522%253A%2520%255B%250A%2520%2520%2520%2520%2520%2520%2522%2523publicKeyModel1Id%2522%252C%250A%2520%2520%2520%2520%2520%2520%2522%2523additional-key%2522%250A%2520%2520%2520%2520%255D%252C%250A%2520%2520%2520%2520%2522keyAgreement%2522%253A%2520%255B%250A%2520%2520%2520%2520%2520%2520%2522%2523publicKeyModel1Id%2522%252C%250A%2520%2520%2520%2520%2520%2520%2522%2523additional-key%2522%250A%2520%2520%2520%2520%255D%252C%250A%2520%2520%2520%2520%2522assertionMethod%2522%253A%2520%255B%250A%2520%2520%2520%2520%2520%2520%2522%2523additional-key%2522%250A%2520%2520%2520%2520%255D%252C%250A%2520%2520%2520%2520%2522capabilityInvocation%2522%253A%2520%255B%250A%2520%2520%2520%2520%2520%2520%2522%2523additional-key%2522%250A%2520%2520%2520%2520%255D%252C%250A%2520%2520%2520%2520%2522capabilityDelegation%2522%253A%2520%255B%250A%2520%2520%2520%2520%2520%2520%2522%2523additional-key%2522%250A%2520%2520%2520%2520%255D%250A%2520%2520%257D%252C%250A%2520%2520%2522didDocumentMetadata%2522%253A%2520%257B%250A%2520%2520%2520%2520%2522canonicalId%2522%253A%2520%2522did%253Asidetree%253AEiDyOQbbZAa3aiRzeCkV7LOx3SERjjH93EXoIM3UoN4oWg%2522%252C%250A%2520%2520%2520%2520%2522method%2522%253A%2520%257B%250A%2520%2520%2520%2520%2520%2520%2522published%2522%253A%2520true%252C%250A%2520%2520%2520%2520%2520%2520%2522recoveryCommitment%2522%253A%2520%2522EiBfOZdMtU6OBw8Pk879QtZ-2J-9FbbjSZyoaA_bqD4zhA%2522%252C%250A%2520%2520%2520%2520%2520%2520%2522updateCommitment%2522%253A%2520%2522EiDOrcmPtfMHuwIWN6YoihdeIPxOKDHy3D6sdMXu_7CN0w%2522%250A%2520%2520%2520%2520%257D%250A%2520%2520%257D%250A%257D)

Comment (OR13) [reply packages/core/src/DocumentComposer.ts:150]: upstream is correct, we are wrong.

Comment (gjgd) [reply packages/core/src/DocumentComposer.ts:150]: Yes, we may need + replace the `methodMetadata` string with  `didDocumentMetadata` and regenerate the fixtures

**Review by gjgd — APPROVED** [review]

**Pull request merged**